README.adoc

```diff
@@ -258,12 +327,32 @@ kubernetes.yaml
 include::finish/kubernetes.yaml[]
 ----
 
-This file defines four {kube} resources. It defines two deployments and two services. A {kube} deployment is a resource responsible for controlling the creation and management of pods. A service exposes your deployment so that you can make requests to your containers. Three key items to look at when creating the deployments are the [hotspot=labels1 hotspot=labels2 hotspot=labels3 hotspot=labels4]`labels`, [hotspot=image1 hotspot=image2]`image`, and [hotspot=containerPort1 hotspot=containerPort2]`containerPort` fields. The [hotspot=labels1 hotspot=labels2 hotspot=labels3 hotspot=labels4]`labels` is a way for a {kube} service to reference specific deployments. The [hotspot=image1 hotspot=image2]`image` is the name and tag of the Docker image that you want to use for this container. Finally, the [hotspot=containerPort1 hotspot=containerPort2]`containerPort` is the port that your container exposes for purposes of accessing your application.
+This file defines four {kube} resources. It defines two deployments and two services. 
+A {kube} deployment is a resource responsible for controlling the creation and management of pods. 
+A service exposes your deployment so that you can make requests to your containers. 
+Three key items to look at when creating the deployments are the [hotspot=labels1 hotspot=labels2 hotspot=labels3 hotspot=labels4]`labels`, 
+[hotspot=image1 hotspot=image2]`image`, and [hotspot=containerPort1 hotspot=containerPort2]`containerPort` fields. 
+The [hotspot=labels1 hotspot=labels2 hotspot=labels3 hotspot=labels4]`labels` is a way for a {kube} service to reference specific deployments. 
+The [hotspot=image1 hotspot=image2]`image` is the name and tag of the Docker image that you want to use for this container. 
+Finally, the [hotspot=containerPort1 hotspot=containerPort2]`containerPort` is the port that your container exposes for purposes of accessing your application.
 For the services, the key point to understand is that they expose your deployments.
-The binding between deployments and services is specified by the use of labels -- in this case the [hotspot=app1 hotspot=app2 hotspot=app3 hotspot=app4 hotspot=app5 hotspot=app6 hotspot=app7 hotspot=app8]`app` label.
+The binding between deployments and services is specified by the use of labels -- in this case the 
+[hotspot=app1 hotspot=app2 hotspot=app3 hotspot=app4 hotspot=app5 hotspot=app6 hotspot=app7 hotspot=app8]`app` label.
 You will also notice the service has a type of [hotspot=NodePort1 hotspot=NodePort2]`NodePort`.
 This means you can access these services from outside of your cluster via a specific port.
-In this case, the ports will be `31000` and `32000`, but it can also be randomized if the [hotspot=nodePort1 hotspot=nodePort2]`nodePort` field is not used.
+In this case, the ports will be `31000` and `32000`, but it can also be randomized if the 
```

Comment:
In this case, the ports will be `31000` and `32000`, but it can also be randomized if the 
--->
In this case, the ports are `31000` and `32000`, but port numbers can also be randomized if the